README.md

```diff
@@ -15,6 +15,25 @@ Use the following instructions to install Codewind with your choice of editor:
 2. [Eclipse plugin](https://github.com/eclipse/codewind-eclipse)
 3. [Eclipse Che plugin](https://github.com/eclipse/codewind-che-plugin)
 
+## Feedback and Community
+
+Have any questions or commments on Codewind? Or would like to get involved in some way? Here's how you can get in touch with us:
+- **Support:** You can ask questions, report bugs, and request features using [GitHub issues](https://github.com/eclipse/codewind/issues).
+- **Public Chat:** Join the public [eclipse-codewind](https://mattermost.eclipse.org/eclipse/channels/eclipse-codewind) and [eclipse-codewind-dev](https://mattermost.eclipse.org/eclipse/channels/eclipse-codewind-dev) Mattermost channels
+- **Twitter:** [@EclipseCodewind](https://twitter.com/EclipseCodewind)
+- **Mailing List:** [codewind-dev@eclipse.org](https://accounts.eclipse.org/mailing-list/codewind-dev)
+- **Weekly Meetings:** We hold [weekly calls](https://github.com/eclipse/codewind/wiki/Codewind-Calls) every Tuesday and Thursday at 9:00 AM EST / 2:00 PM GMT.
```

Comment:
Minor suggestion - take it or leave it: `9:00 AM EST / 2:00 PM GMT.` -> `9:00AM Eastern Time / 14:00 UTC`

Reply:
I'm good with that